test/FunctionalTests/Microsoft.OData.Core.Tests/UriParser/Parsers/SelectExpandParserTests.cs

```diff
@@ -298,6 +298,24 @@ public void ParseNestedOrderByAndSearchInSelectWorks()
             selectTermToken.SearchOption.ShouldBeStringLiteralToken("xyz");
         }
 
+        [Fact]
+        public void ParseOrderByThisInSelectWorks()
+        {
+            // Arrange & Act
+            SelectToken selectToken = ParseSelectClause("Emails($orderby=$this)");
+
+            // Assert
+            Assert.NotNull(selectToken);
+            SelectTermToken selectTermToken = Assert.Single(selectToken.SelectTerms);
+            selectTermToken.PathToProperty.ShouldBeNonSystemToken("Emails");
+            Assert.NotNull(selectTermToken.OrderByOptions);
+            OrderByToken orderBy = Assert.Single(selectTermToken.OrderByOptions);
+            orderBy.Expression.ShouldBeRangeVariableToken("$this");
+
+            // If you don't specify direction, it's Ascending by default.

```

Comment:
>// If you don't specify direction, it's Ascending by default. [](start = 12, length = 61)

you can expand [Fact] to [InlineData] to test 
1) doesn't specify the direction
2) specify the ascend direction
3) specify the descend direction

and get rid of this comment

Reply:
#Resolved